server/hedley/modules/custom/hedley_admin/hedley_admin.module

```diff
@@ -416,3 +424,184 @@ function hedley_admin_administrators_access() {
 
   return user_has_role($admin_role->rid);
 }
+
+/**
+ * Implements hook_form() for generating the Resilience survey results table.
+ *
+ * This function generates a table displaying the resilience survey results.
+ */
+function hedley_admin_resilience_survey_results_table(array $form, array &$form_state) {
+  $header = array(
+    'nurse' => t('Nurse'),
+    'health_center' => t('Health Center'),
+    'survey_type' => t('Survey Type'),
+    'date_measured' => t('Date Measured'),
+  );
+
+  for ($i = 1; $i <= 12; $i++) {
+    $header["question_$i"] = t('Q@number', ['@number' => $i]);
+  }
+
+  $query = new EntityFieldQuery();
+  $query->entityCondition('entity_type', 'node')
+    ->entityCondition('bundle', 'resilience_survey')
+    ->propertyCondition('status', 1);
+
+  $result = $query->execute();
+
+  if (empty($result['node'])) {
+    drupal_set_message(t('No survey results available.'), 'warning');
+    return [];
+  }
+
+  $survey_nids = array_keys($result['node']);
+  $survey_nodes = node_load_multiple($survey_nids);
```

Comment:
I don't think see this was implemented:

![Selection_1189](https://github.com/user-attachments/assets/1c9fc6ed-684b-44d3-a78c-520dffd25c97)

If not done, when there are too many results, we'll get out of memory exception.